tests/test_training.py

```diff
@@ -45,39 +45,72 @@ def data_dir():
     return DataDir("test_training")
 
 
+@pytest.fixture(params=["ru", "zh"])
+def trg_lang(request):
+    return request.param
+
+
+@pytest.fixture
+def config(trg_lang):
+    zh_config_path = os.path.abspath(os.path.join(FIXTURES_PATH, "config.pytest.enzh.yml"))
+    return zh_config_path if trg_lang == "zh" else None
+
+
 @pytest.fixture()
-def vocab(data_dir):
-    shutil.copyfile("tests/data/vocab.spm", data_dir.join("vocab.spm"))
+def vocab(data_dir, trg_lang):
+    output_path = data_dir.join("vocab.spm")
+    if trg_lang == "ru":
+        shutil.copyfile("tests/data/vocab.spm", output_path)
+    else:
+        shutil.copyfile("tests/data/vocab.zhen.spm", output_path)
+
+    return output_path
 
 
 @pytest.fixture()
-def corpus(data_dir):
+def corpus(data_dir, trg_lang):
+    sample = zh_sample if trg_lang == "zh" else ru_sample
     data_dir.create_zst("corpus.en.zst", en_sample)
-    data_dir.create_zst("corpus.ru.zst", ru_sample)
+    data_dir.create_zst(f"corpus.{trg_lang}.zst", sample)
+    data_dir.create_zst("mono.en.zst", en_sample)
+    data_dir.create_zst(f"mono.{trg_lang}.zst", sample)
     data_dir.create_zst("devset.en.zst", en_sample)
-    data_dir.create_zst("devset.ru.zst", ru_sample)
+    data_dir.create_zst(f"devset.{trg_lang}.zst", sample)
 
 
 @pytest.fixture()
-def alignments(data_dir, vocab, corpus):
+def alignments(data_dir, vocab, corpus, trg_lang, config):
     env = {
         "TEST_ARTIFACTS": data_dir.path,
         "BIN": bin_dir,
         "MARIAN": marian_dir,
         "SRC": "en",
-        "TRG": "ru",
+        "TRG": trg_lang,
     }
-    data_dir.run_task("alignments-original-en-ru", env=env)
-    shutil.copyfile(
-        os.path.join(data_dir.path, "artifacts", "corpus.aln.zst"), data_dir.join("corpus.aln.zst")
-    )
+    for task, corpus in [("original", "corpus"), ("backtranslated", "mono")]:
```

Comment:
Question: Can these be pytest parameters instead? Or do they need to run sequential. If they can be parameters it would make it easier to re-run the tests and isolate failures.

Reply:
It's not a test, it's a fixture that produces alignments for the original and back-translated corpus.